clients/python_client/Pipfile

```diff
@@ -15,4 +15,4 @@ tox = "*"
 pre-commit = "*"
 
 [requires]
-python_version = "3.9"
+python_version = "3"
```

Comment:
https://github.com/pypa/pipenv/issues/1050 `pipenv` has declined to support `PEP 508 specifiers`